source/reference/beta-program.txt

```diff
@@ -12,13 +12,70 @@
    :depth: 2
    :class: singlecol
 
-Content for this page will be created in DOCSP-38285.
+.. include:: /includes/beta-program-intro.rst
+
+Each ``mongosync`` release will have a corresponding {+c2c-beta-program-short+} 
+build that includes its own set of experimental features.
+
+Get Started 
+-----------
+
+To request access to the {+c2c-beta-program-short+} binaries, please file a 
+`HELP ticket <https://jira.mongodb.org/browse/HELP>`_.
+
+.. warning:: Disclaimer
+
+   MongoDB restricts access to beta features via the beta build. Your use of the 
+   beta is governed by the language specified in the 
+   `Cloud Subscription Agreement <https://www.mongodb.com/cloud-subscription-agreement/november-2023>`_, 
+   `Cloud Terms of Service <https://www.mongodb.com/legal/terms-and-conditions/cloud>`_, 
+   or other applicable agreement between you and MongoDB with respect to your 
+   use of ``mongosync``.
+   
+   You understand that features are available through the beta build, and flags 
+   are not generally available. All use of beta builds is at your own risk. 
+   Beta builds provide no stability guarantees; API, UI, options, and behaviors 
+   may change or be removed any time. MongoDB will never issue a Critical 
+   Advisory or notify about defects in beta builds. 
+   
+   You should follow feature usage guidance provided by Field and R&D without 
+   deviation and should only use beta builds for one-time, human-supervised 
+   migrations (e.g., not for continuous sync or automated migrations). Beta 
+   capabilities are not intended for use to migrate production workloads.
 
 Beta Features
 -------------
 
+{+c2c-beta-program-short+} {+version-dev+} includes the following features:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 30 70
+
+   * - Feature
+     - Description
+
+   * - :ref:`c2c-beta-namespace-remapping`
+     - # DESCRIPTION TBD 
```

Comment:
[question]

Should these descriptions still be TBD? Thanks!

Reply:
Yes! The corresponding tickets for the ones that are TBD haven't been completed yet. We'll be doing one more sweep through on [DOCSP-39992](https://jira.mongodb.org/browse/DOCSP-39992) to update any missing descriptions and make sure everything's covered.